### PR TITLE
docs: setup finetuner documentation

### PR DESCRIPTION
**Pull request by bwanglzu**
Created the basic setup of documentation, including index and documentation structure (content not filled in yet).

It should be noted that document structure should also includes guides (e.g. finetuner resnet50 on TLL) and developer reference (auto-generated docstring). Please take over from here

C481FAF3-265A-4767-8D6A-492765DFAC6A

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,17 +21,17 @@ jobs:
...
-      - uses: actions/checkout@v2
-      - name: Set up Python 3.7
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.7
-      - name: Install requirements
-        run: make init
-      - name: Run tests
-        run: make test
+#  run_tests:
```
Comment (gmastrapas) [inline .github/workflows/ci.yml:24]: why is this commented out?

**Comment by jupyterjazz** [comment]
> Docs are deployed on https://ft-docs-setup-finetuner-docs--jina-docs.netlify.app

I think it deployed the old docs for some reason..

**Comment by gmastrapas** [comment]
> > Docs are deployed on https://ft-docs-setup-finetuner-docs--jina-docs.netlify.app
> 
> I think it deployed the old docs for some reason..

A while back we had the chance to select the version. By default you are seeing the main branch. Try https://ft-docs-setup-finetuner-docs--jina-docs.netlify.app/docs-setup-finetuner-docs/

File: .github/workflows/label-pr.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+name: PR
+
+on:
+  pull_request:
+
+jobs:
+  assign-label-to-pr:
+    runs-on: ubuntu-latest
```
Comment (jupyterjazz) [inline .github/workflows/label-pr.yml:8]:
```suggestion
  assign-label-to-pr:
    name: Assign label to PR
    runs-on: ubuntu-latest
```

File: .github/workflows/label-pr.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+      - id: docs_updated
+        if: contains( github.event.pull_request.labels.*.name, 'area/docs')
+        run: echo '::set-output name=docs::true'
+    outputs:
+      docs: ${{ steps.docs_updated.outputs.docs }}
+
+  deploy-to-netlify:
+    runs-on: ubuntu-latest
```
Comment (jupyterjazz) [inline .github/workflows/label-pr.yml:29]:
```suggestion
  deploy-to-netlify:
    name: Deploy to netlify
    runs-on: ubuntu-latest
```

**Review by jupyterjazz — COMMENTED** [review]
minor changes to make naming consistent

**Review by jupyterjazz — APPROVED** [review]
LGTM!

**Pull request merged**